scripts/ci/run_perftest.sh

```diff
@@ -233,6 +233,9 @@ query_types() {
     metaquery)
       echo field-keys tag-values
       ;;
+    iot)
+      echo aggregate-keep light-level-8-hr sorted-pivot
```

Comment:
Could you also add `1-home-12-hours` while you're here? And make the line match what's on `master` to help avoid any conflicts for future backports:
```suggestion
      echo 1-home-12-hours light-level-8-hr aggregate-keep sorted-pivot
```